README.md

```diff
@@ -89,16 +95,26 @@ Check out `SnowflakeIngestBasicExample.java`
 ## Snowpipe Streaming
 
 Check out `SnowflakeStreamingIngestExample.java`, which performs following operations:
-1. Reads a JSON file which contains details regarding Snowflake Account, User, Role and Private Key. Take a look at `profile_streaming.json.example` for more details.
-    1. [Here](https://docs.snowflake.com/en/user-guide/key-pair-auth.html#configuring-key-pair-authentication) are the steps required to generate a private key.
-2. Creates a `SnowflakeStreamingIngestClient` which can be used to open one or more Streaming Channels pointing to the same or different tables.
+
+1. Reads a JSON file which contains details regarding Snowflake Account, User, Role and Private Key. Take a look at
+   `profile_streaming.json.example` for more details.
+    1. [Here](https://docs.snowflake.com/en/user-guide/key-pair-auth.html#configuring-key-pair-authentication) are the
+       steps required to generate a private key.
+2. Creates a `SnowflakeStreamingIngestClient` which can be used to open one or more Streaming Channels pointing to the
+   same or different tables.
 3. Creates a `SnowflakeStreamingIngestChannel` against a Database, Schema and Table.
-    1. Please note: The database, schema and table is expected to be present before opening the Channel. Example SQL queries to create them:
+    1. Please note: The database, schema and table is expected to be present before opening the Channel. Example SQL
+       queries to create them:
+
 ```sql
-create or replace database MY_DATABASE;
-create or replace schema MY_SCHEMA;
-create or replace table MY_TABLE(c1 number);
+create
+or replace database MY_DATABASE;
+create
+or replace schema MY_SCHEMA;
+create
+or replace table MY_TABLE(c1 number);
```

Comment:
The formatting here seems wrong

Reply:
this is weird, let me see why auto formatter decided to do this